src/test/suite/asciidocUtilsTest.test.ts

```diff
@@ -42,4 +49,18 @@ suite('AsciiDoc Utils Test Suite', () => {
 		expect(content).to.include('The quick brown fox jumps over the lazy dog.');
 	});
 
+	test('get the adoc parser', () => {
+		expect(adocutils.getADParser()).to.not.be.null;
+	});
+
+	test('that we can parse adoc to html', () => {
+		const htmlOutput = adocutils.parseADtoHTML(BASIC_EXAMPLE);
+		expect(htmlOutput).to.include(`<h1>Document Title</h1>`);
+		expect(htmlOutput).to.include(`<p>That&#8217;s all, folks!</p>`);
+
+		// this is tested more effectively with 'open an asciidoc file with an include' above
+		const htmlOutputWithBaseDir = adocutils.parseADtoHTML(BASIC_EXAMPLE, '');
+		expect(htmlOutput).to.include(`<h1>Document Title</h1>`);
+		expect(htmlOutputWithBaseDir).to.include(`<p>That&#8217;s all, folks!</p>`);
```

Comment:
this sounds like it can be splitted in 2 tests; I will allow to have a better view of what is tested and also if th efirst case is failing, it will allow to kno wif the second is failing too.